recipes/pyseas/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set version = "3.0.0" %}
+
+package:
+  name: pyseas
+  version: {{ version }}
+
+source:
+  fn: pyseas-{{ version }}.tar.gz
+  url: https://bitbucket.org/ooicgsn/pyseas/get/{{ version }}.tar.gz
+  sha256: dd317eee2c0f61d2791d1c3ff381502e34b1708fd549f2888b12ccddc43b2b89
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - geomag
+    - gsw
```

Comment:
Looking at the code it seems that `pyseas` uses `pygsw` and now `gsw`. Can @cwingard or @joefutrelle confirm that?
If so we need to package `pygsw` in conda-forge. (We do not have it at the moment b/c it is slow (vectorized from c), outdated, and hard to build.)